HACKING.md

```diff
@@ -221,8 +221,16 @@ SELinux type.
     $ sudo make install
     $ sudo cp src/bridge/cockpit.pam.insecure /etc/pam.d/cockpit
```

Comment:
I think you wanted to delete that line?

Reply:
*cough* *cough*, I did! /me whistles innocently..